biometrics/extract.py

```diff
@@ -213,6 +213,9 @@ def _pileup(self, bam, site):
                     base_qual = pileupread.alignment.qual[pileupread.query_position]
                 except:
                     base_qual = 30
+                 #to deal with non-ascii characters
+                if base_qual == '#':
```

Comment:
I would do any string and add a default value of 30 as done above instead of continue